`zig reduce` fails to compile

### Zig Version

0.15.0-dev.532+a3693aae3

### Steps to Reproduce and Observed Behavior

Just run `zig reduce`. I even don't know what it does but it at lease launches on stable `0.14.0`

```
/home/dustdfg/Downloads/zig-linux-x86_64-0.15.0-dev.532+a3693aae3/lib/compiler/reduce.zig:406:9: error: expected type 'mem.Alignment', found 'comptime_int'
        1,
        ^
/home/dustdfg/Downloads/zig-linux-x86_64-0.15.0-dev.532+a3693aae3/lib/std/mem.zig:22:23: note: enum declared here
pub const Alignment = enum(math.Log2Int(usize)) {
                      ^~~~
/home/dustdfg/Downloads/zig-linux-x86_64-0.15.0-dev.532+a3693aae3/lib/std/fs/Dir.zig:1980:32: note: parameter type declared here
    comptime alignment: std.mem.Alignment,
                        ~~~~~~~^~~~~~~~~~
referenced by:
    main: /home/dustdfg/Downloads/zig-linux-x86_64-0.15.0-dev.532+a3693aae3/lib/compiler/reduce.zig:123:25
    posixCallMainAndExit: /home/dustdfg/Downloads/zig-linux-x86_64-0.15.0-dev.532+a3693aae3/lib/std/start.zig:671:37
    4 reference(s) hidden; use '-freference-trace=6' to see all references
```

### Expected Behavior

It compiles...

--- a/lib/compiler/reduce.zig
+++ b/lib/compiler/reduce.zig
@@ -403,7 +403,7 @@ fn parse(gpa: Allocator, file_path: []const u8) !Ast {
         file_path,
         std.math.maxInt(u32),
         null,
-        1,
+        .fromByteUnits(1),
         0,
     ) catch |err| {
         fatal("unable to open '{s}': {s}", .{ file_path, @errorName(err) });

